workflows/edit_data_packages/add_a_pre-generated_identifier.Rmd

```diff
@@ -0,0 +1,13 @@
+
+## Add a pre generated identifier to the eml
+
+When you pre generate a UUID or DOI, the change is not automatically reflected in the "Identifier" section of the eml so this makes sure that the eml lines up with the identifier being used. 
+
+```{r, eval = FALSE}
+## Generate DOI and add to EML
+doiPid <- dataone::generateIdentifier(mn, "DOI")
+eml@packageId <- new("xml_attribute",
+                      .Data = doiPid)
+```
+
+Be sure to include the `identifier=` argument in your [publish update](#publish-update) command so the pre generated identifier is applied.
```

Comment:
Again needs more context here. You usually don't want to publish the pre-generated id right away. Typically you would do it after several commits once the package is finalized.